### ISSUE TITLE
Choco Switch Support

### ISSUE THREAD

**Issue by upsampled** [swill/kad#8]
I am trying to add Choco Switch support to 'kad'. The plate cutouts should be the same as normal switches, but they have different keycap dimensions. The low-profile, standard keycaps used by the Choco Switches appear to not be perfectly square. At a minimum this means I need to replace U1 with U1.Width and U1.Height and corresponding math to allow a non-square U1.

Before I dive in, can you think of any other logic that might get thrown off by this change?

Naturally I'll submit anything that works as a PR and leave it up to you to add it. 

Low Profile 1u difference
1)  17.5mm x 18.5mm - https://oddrocketkeyboards.blogspot.com/2018/05/koda-keyboard-4x12-grid-using-kailh-lp.html
2) 17.5mm x 16.5mm -  https://www.reddit.com/r/MechanicalKeyboards/comments/7hjks9/help_dimensions_of_low_profile_keycaps/
https://www.reddit.com/r/MechanicalKeyboards/comments/7h79qg/comment/dqp63tw

**Comment by upsampled** [comment]
Actually, let me order some keycaps and confirm this. I am seeing schematics still spacing them at 1U standard.

**Comment by upsampled** [comment]
Just found out first hand that I will actually need to make a new cutout as well. Confirming this thread:
 
https://www.reddit.com/r/MechanicalKeyboards/comments/79cbez/helpdoes_kalih_low_profile_swithes_fit_cherry_mx/

**Comment by swill** [comment]
Thanks for confirming.  I suspected that they would work as expected, but please report back if you have any issues.  Cheers...

**Comment by upsampled** [comment]
@swill sorry, I used this ticket to collect notes. Now that i made a test plate let me summarize my findings:

1. `kad` currently cannot handle the [low profile keycaps](https://novelkeys.xyz/collections/keycaps/products/kailh-low-profile-keycaps-with-legends) because they require a non-square 1U. I added this capability in PR #9 . 
2. The square 14mm by 14mm cutout will work for the choco switches. (I probably will reduce the cutout slightly for a more snug fit later).
3. Material thickness must be less than 1.3mm for clips to function. (thickness is used in the code but worth mentioning)

**Test Plate**
![image](https://user-images.githubusercontent.com/34343319/60906663-a8350180-a245-11e9-9421-13687e79cbf5.png)
